R/benchmarks.R

```diff
@@ -41,9 +42,10 @@ test.convergence = function(row.nr=2, convergence.iterations = 20, iterations =
   }
   if(return.value == "values") {
     if(getTaskType(task) == "classif"){
-      result = plyr::count(as.data.frame(values))
-      result$values = as.character(result$values)
-      result$values[match(prediction.class, result$values)] = paste0(">>", prediction.class, "<<")
+      values = replace(values, grep(prediction.class, values), paste0(">>", prediction.class, "<<"))
+      result = as.data.frame(values)
+      names(result) = "values"

```

Comment:
Ich habe herausgefunden, dass man bei `as.data.frame` den parameter `row.names` übergeben kann. Damit können wir uns diese Zeile hier sparen. Und an mindestens 4 Stellen in meinem Code auch :)